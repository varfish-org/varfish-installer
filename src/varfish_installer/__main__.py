import sys

from loguru import logger

from varfish_installer import cmd_download  # noqa: F401
from varfish_installer import cmd_install  # noqa: F401
from varfish_installer.cli import app

if __name__ == "__main__":
    logger.add(sys.stderr, format="{time} {level} {message}", level="DEBUG")
    app()
