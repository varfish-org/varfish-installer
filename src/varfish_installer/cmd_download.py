"""Implementation of the ``download`` command."""

import pathlib

from loguru import logger

from varfish_installer.cli import app
from varfish_installer.config import DownloadConfig, DownloadDataset, DownloadSubset

#: Default configuration to use.
DEFAULT_CONFIG = DownloadConfig()


@app.command()
def install(
    dst_path: pathlib.Path,
    base_dir: str = DEFAULT_CONFIG.base_dir,
    static_infix: str = DEFAULT_CONFIG.static_infix,
    subset: DownloadSubset = DEFAULT_CONFIG.subset,
    disable_ssl_verification: bool = DEFAULT_CONFIG.disable_ssl_verification,
    datasets: list[DownloadDataset] = DEFAULT_CONFIG.datasets,
):
    """Update data downloads at the given path."""
    config = DownloadConfig(
        base_dir=base_dir,
        static_infix=static_infix,
        subset=subset,
        disable_ssl_verification=disable_ssl_verification,
        datasets=datasets,
    )
    logger.debug(f"Downloading data to {dst_path} with config {config}")
