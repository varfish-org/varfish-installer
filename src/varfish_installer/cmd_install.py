"""Implementation of the ``install`` command."""

import pathlib
from typing import Optional

from loguru import logger

from varfish_installer.cli import app
from varfish_installer.config import InstallConfig, TlsCertificateMode

#: Default configuration to use in CLI.
DEFAULT_CONFIG = InstallConfig()


@app.command()
def install(
    dst_path: pathlib.Path,
    manage_docker_compose_override: bool = DEFAULT_CONFIG.manage_docker_compose_override,
    manage_env: bool = DEFAULT_CONFIG.manage_env,
    varfish_web_dns_names: list[str] = DEFAULT_CONFIG.varfish_web_dns_names,
    traefik_tls_mode: TlsCertificateMode = DEFAULT_CONFIG.traefik_tls_mode,
    traefik_tls_cert_file: Optional[str] = DEFAULT_CONFIG.traefik_tls_cert_file,
    traefik_tls_key_file: Optional[str] = DEFAULT_CONFIG.traefik_tls_key_file,
):
    """Create VarFish installation at the given path."""
    config = InstallConfig(
        manage_docker_compose_override=manage_docker_compose_override,
        manage_env=manage_env,
        varfish_web_dns_names=varfish_web_dns_names,
        traefik_tls_mode=traefik_tls_mode,
        traefik_tls_cert_file=traefik_tls_cert_file,
        traefik_tls_key_file=traefik_tls_key_file,
    )
    logger.debug(f"Installing VarFish at {dst_path} with config {config}")
    logger.debug(
        f"git clone https://github.com/varfish-org/varfish-docker-compose-ng.git {dst_path}"
    )
