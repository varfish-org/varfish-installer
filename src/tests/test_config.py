from varfish_installer.config import DownloadConfig

#: Default configuration to use.
DEFAULT_CONFIG = DownloadConfig()


def test_smoke():
    pass
