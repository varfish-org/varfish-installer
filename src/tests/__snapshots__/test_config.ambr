# serializer version: 1
# name: test_manifest_parser
  Manifest(raw_content='## Invoked from: /data/cephfs-1/work/projects/cubi-varfish-data/2023-06-05_varfish-db-downloader/varfish-db-downloader/output/reduced-exomes/annonars/gnomad-exomes-grch38-4.0+0.33.0\n## $ hashdeep -l -r .\n## \n%%%% HASHDEEP-1.0\n1544,bce1f589f031405ab14c6edec1c4f1ee,49fa6395c7046c5c5e9a45466dad0d5870fbfad4bdbb527766197ef4d9f0fb26,./rocksdb/000372.sst\n1089218854,0ef83d014fa784cc5cf420ec22f84301,040fb94cfcf6c9dd74fc36e7b2d2f58eb2b03bf7852210ef18156cca8f699fbb,./rocksdb/000375.sst\n1088933980,c22221b854d45167287593a872d00fcf,e2a10ec40a5d50c5c1daf1e1152bc47d93e7c634c62b808403511d31adac14b5,./rocksdb/000376.sst\n1088792107,16065f630ff3f16a4a3ee8c952643d3b,5a88839309e5bd516495d63f2dfc703cddc3e570b4f4651c297c39b991d0476c,./rocksdb/000377.sst\n1089230569,d735b412fcca1d9af9819ba74904c2c9,71d54244b7e64c9917f851e49adf099ea336f249e969a93469fba2de964b4a53,./rocksdb/000378.sst\n1089212984,c7b59ff5730a0513333319c38d8c6a4a,01ab3e8858f05e500859555b14c23725aaa4542ceb49e87b53012a95e55c9b36,./rocksdb/000379.sst\n1088994759,3b5533977d96cc63f522fb2db36f6d73,ce3c8d710fcc5b9757e168ed1857cd65434925e71a4e143e14fa702ef877679d,./rocksdb/000380.sst\n1089077399,d6a5578f60d36b5ea24e24fe23cb27b3,f02b569f62a8d161a322625ba08b143bfb3ed4e19c8fead1ab6719747360c253,./rocksdb/000381.sst\n1089017411,d428ab22dda7a2a40789ebc6c1da7610,4b14183a984a7d6d71fe353a47f1eb2c770baef152a5af55943d351d0a753a4a,./rocksdb/000382.sst\n1089188918,105ba3e300a1d5b6c827ef6f55d3b7ed,30c2e153ba87da94897f01d0a21f1f5399700e07c63ead911cb1c5203a1ef276,./rocksdb/000383.sst\n1089056095,94918cd025eb83cdc84bad6b507e0472,9a4213c8e3fee07c0573c17b5a863d777ad67f235d7a35444bd66379142cffff,./rocksdb/000384.sst\n1088813782,41eac3bb8c3bc97902ce35503d257b9c,d73abdcc446ee53bd6d5d1f27b1bc9921d400698935f235e35bc2bf663c7684e,./rocksdb/000385.sst\n1088738712,5d563b68c08ab3836e5ae3affd3d9f7e,eadd9cc501fcb805fb87be6757cae7e2f6765f06015b91b4529f34b44fe6376b,./rocksdb/000386.sst\n1089106381,28ecc06fce9f7798a223e0dd2d57cb10,7cae5221dbae31dfc59046a64744210a1b696cb3128d7e7dadc4b65ec319a48f,./rocksdb/000387.sst\n1089262649,e828f9d191d8941a3bd08ce88fc31f68,2da9074240ddca4786e913a8396963b1348f8714dff5d5fb05de26a5427a3a4c,./rocksdb/000388.sst\n1089302816,9752708bbbc80821d189fe36aa2bac4c,f1d68d1877c1b0353e23849a20c66c6a093801c18f60a6a3aa383402c06daca1,./rocksdb/000389.sst\n1088110834,44da39af9377477f119c4c7e722cf5c2,78228d63521281e575a92ad76116be2cf035e23dc96ab6b950fe81c43d5fda88,./rocksdb/000390.sst\n1089346071,ffb30dd2f5b6a477a745272f940d4c05,7d65d5224f1c487977e7f5d6752b69c50cf0791eb066464cc1ab147c41bb317a,./rocksdb/000391.sst\n1089013161,db805eda066735401cfa35c49892ffad,665bc665cd50ee5da39e39767ab835e59d9077f10e5e7957b7cdce60bad6635a,./rocksdb/000392.sst\n1088972734,95b1147f1fd0fadec7185759c4f05883,0501794cbd24949f0fd404a463291540b4bae30a8fc511cce9cc9c735672da81,./rocksdb/000393.sst\n1089048629,511c38c6497a2ee21f5dd6ab6c32fe3c,bc23c9dae5d7fab6188a812001923c67b6bc80c1b6c18227639c099f50549249,./rocksdb/000394.sst\n1089284704,a70f9410b8d3e547174e747c1ed67866,b66f5a36654fdce09c538e3311e961e8c182be5dad3e876506f6048625fdc48c,./rocksdb/000395.sst\n1088982929,a01527af1e94c731a7ec4eda10dca40f,cc46c8290d40899f08194a471b0acfaa452e55e2f3e47747fdc9e203847d3003,./rocksdb/000396.sst\n1089130285,6b40b5a7af7e7345aab723d4e1cce064,82d2a3ed3ec01b2b18e418bcf73ec3c4498b06566a2bb08e5af1920bb3f0474e,./rocksdb/000397.sst\n1088976586,0dee0be71fe191f4f412fafe49409c83,c81a00fe0a01e427f0bdd5dff58107cf595388c5c9d79b9e077e726dc911f675,./rocksdb/000398.sst\n1088855396,278f0d888025a90d16672833abe81024,819ceaa0d29f5a0cd3084c1e1998247b2f8973c97fc64bc84d0903064df38853,./rocksdb/000399.sst\n1089260009,e9832468d4353076c5c64195cfdab763,0185c3dab3c5c7b9c7e97321bc50b64872914f247f6e2d612d7bd0b2f6112699,./rocksdb/000400.sst\n1089257638,c14092fa71dd9230acadf1834c729f2d,8619ad6b9ed4bac9f371d458a848e0584c5722248d68d8c9e1f532c43213697f,./rocksdb/000401.sst\n1089174502,cc55451980cbc4e1a03e69df4addcdae,1ac43b27b2052c3c86466178628cb69c0d545b8e0f6de7209383274abe8d8a01,./rocksdb/000402.sst\n1089236809,2ba7a96e58c0c4e3ec2fd1cb1a1fce44,2109784ca6bd3ef8d8b767d304df2dc046029e2f672a549553ade2a0216c3315,./rocksdb/000403.sst\n1089136948,25f96529f975fcc5faae52928bb46d6e,854038cd91778e9a0c237938379c1a832d6a799030ff9bd7503fe8ff9ceaa14f,./rocksdb/000404.sst\n1089294821,a48930cce95be74fc9e719de6506fa5c,5b815c2de5edac74d121be71f3e0df90083c78290413deec47504a39dd3fbb87,./rocksdb/000405.sst\n1089247949,6291b6a5cf3f9c216e35b582ae82d389,572ce0ab8fae1b1a3fb95d37b2fe1395aff68837fb7d7b008fb4d2d2fdda6fdf,./rocksdb/000406.sst\n1089046034,38dda484748b378eae5d48a44ffc8936,45e0dc396b19b75c7f64985373e920e22506205c8835d4a352465b61ba3d539c,./rocksdb/000407.sst\n1089138528,61715bedec63061261485b583bf54d55,a3425e6dbabec000f790ec738e446046be6e868044d2ca92104a48bbe4ca8b7e,./rocksdb/000408.sst\n1088961607,59ac19d1f3bf2f84f208454fcd5ec60f,693617869c19fbedbffa6832138b6dd661591e54e3fabea3a5363ce43e4d1017,./rocksdb/000409.sst\n1089276807,62319ef6607ddc45b9d668140c8c7d36,6ffddafdb31ca481c6eace1c744f03af143d0c474c50b01739f993efaab4530e,./rocksdb/000410.sst\n1088962325,7fb206d657d7b2382755fc1b8538ef5a,f4a3584ff1e9487ebc0fe9945014f307b2c4d0550da8acbeffa5ab5a4b1da1bc,./rocksdb/000411.sst\n1089115549,c2a14a1a51e8a5540647dcec3947befe,22cf6bb0042d4b5e0833715d53c8d82acc9ad1b6c917f24e0961bbb02037358f,./rocksdb/000412.sst\n1089093235,1e56706a6a72dc049a7df6ee0b85129a,0e2cb93f1df7d2db461c9789fd103234cab950aa4705f75a80be7dd4fbea9df6,./rocksdb/000413.sst\n1088971425,d2539a82d2864bff326b6aadbff97d34,402519d9a92bba62a3b360ee2a10724a9d52763ce57f77a3f4c0016fd2e560ad,./rocksdb/000414.sst\n1089171186,52cdb06cb054ae9140f4d49488113e48,74dfd3503a2ccf71b4b83077437102be50243bfadc01baf54a2ade49a70fb4d3,./rocksdb/000415.sst\n859126362,a37af6fff97efd0d090a88b5964ddc79,dc5f82e43a20f0a71f46c0718393244dce1ea02d96dc2e5f06a33afba1dd7ce4,./rocksdb/000416.sst\n694644230,04373b1020e2ed5c4fe3ef844f5a743b,8b8ab7838627a90e8cc9e52de5eb5b7668c7686c1741240c3e509e5805aeee52,./rocksdb/000417.sst\n879071264,c82041866827852b3765833fcdb05792,353d33acf7e6fd8f6d6369d72c6abe474b233e11596db30d0e98065f6dacc4f7,./rocksdb/000418.sst\n895176213,4648b340f43c32b8eecd5679ff43ab23,ed0c5056f190b98083ed7844c6b9c894be64e5e8e1d69f4f5a1d2a6f420fea36,./rocksdb/000419.sst\n862698256,150c80987ef8729a00b55ecfa86a4f2a,430544b47b033a75bff5da664bd90ef7eeaf54e09d930cabddf96e168f785cd5,./rocksdb/000420.sst\n884488381,330809257229ed35ada44952c52f4828,bcf01a45623a9e2f6f53410a69681f9b31cbd1f0d0ec956520b27095734cde1a,./rocksdb/000421.sst\n910629924,d7e016dfed4f2726b01b152eee37b41d,590b5090a0aea06ad78a83049fd299d00b245ad7cb30f8b677fcc0eec67d41c9,./rocksdb/000422.sst\n37395883,8d950020bb50e1fc1b4e619f23051208,aab8b81c56b26dfb3641bb689a3d07b5e76b942e23939ed1b55ba29d39a86eea,./rocksdb/000423.sst\n16,9f36605efba98dab15728fe8b5538aa0,9c283f6e81028b9eb0760d918ee4bc0aa256ed3b926393c1734c760c4bd724fd,./rocksdb/CURRENT\n36,053182ada9c50b2d8010935c6da8b1b4,1dc7b40d8645ad5084cf8ff0c483048e78be7739f3861456ca31e42e833b539d,./rocksdb/IDENTITY\n0,d41d8cd98f00b204e9800998ecf8427e,e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855,./rocksdb/LOCK\n1060894,091faf0b2b35346ac07368aaf3b1fae6,dc917ab03b15659b4eec772320478d038e2a68d27dfc77eeadcbc942524a100d,./rocksdb/LOG\n31518,952ef0c700c1f640f95db43d42c2ff61,350e2556fa2bf105c5e8abf3cf7343418d04f90ae11ec4a7a3f92b312d0239eb,./rocksdb/MANIFEST-000005\n15392,25c6dbe82754022c1d0985514cc2c4dd,e1478633f327f9ddfaa608bd11e263bfc45d9fde5b613cdd2a7eefe0251cf486,./rocksdb/OPTIONS-000009\n15392,25c6dbe82754022c1d0985514cc2c4dd,e1478633f327f9ddfaa608bd11e263bfc45d9fde5b613cdd2a7eefe0251cf486,./rocksdb/OPTIONS-000011\n%%%% size,md5,sha256,filename\n## EOF SHA256=96748c099ac0787369bccd3e5879e05ba420011f1f131240beb437deeea42097\n', hashdeep_invocation_path='/data/cephfs-1/work/projects/cubi-varfish-data/2023-06-05_varfish-db-downloader/varfish-db-downloader/output/reduced-exomes/annonars/gnomad-exomes-grch38-4.0+0.33.0', sha256_checksum='96748c099ac0787369bccd3e5879e05ba420011f1f131240beb437deeea42097', entries=[ManifestEntry(size=1544, md5='bce1f589f031405ab14c6edec1c4f1ee', sha256='49fa6395c7046c5c5e9a45466dad0d5870fbfad4bdbb527766197ef4d9f0fb26', name='./rocksdb/000372.sst'), ManifestEntry(size=1089218854, md5='0ef83d014fa784cc5cf420ec22f84301', sha256='040fb94cfcf6c9dd74fc36e7b2d2f58eb2b03bf7852210ef18156cca8f699fbb', name='./rocksdb/000375.sst'), ManifestEntry(size=1088933980, md5='c22221b854d45167287593a872d00fcf', sha256='e2a10ec40a5d50c5c1daf1e1152bc47d93e7c634c62b808403511d31adac14b5', name='./rocksdb/000376.sst'), ManifestEntry(size=1088792107, md5='16065f630ff3f16a4a3ee8c952643d3b', sha256='5a88839309e5bd516495d63f2dfc703cddc3e570b4f4651c297c39b991d0476c', name='./rocksdb/000377.sst'), ManifestEntry(size=1089230569, md5='d735b412fcca1d9af9819ba74904c2c9', sha256='71d54244b7e64c9917f851e49adf099ea336f249e969a93469fba2de964b4a53', name='./rocksdb/000378.sst'), ManifestEntry(size=1089212984, md5='c7b59ff5730a0513333319c38d8c6a4a', sha256='01ab3e8858f05e500859555b14c23725aaa4542ceb49e87b53012a95e55c9b36', name='./rocksdb/000379.sst'), ManifestEntry(size=1088994759, md5='3b5533977d96cc63f522fb2db36f6d73', sha256='ce3c8d710fcc5b9757e168ed1857cd65434925e71a4e143e14fa702ef877679d', name='./rocksdb/000380.sst'), ManifestEntry(size=1089077399, md5='d6a5578f60d36b5ea24e24fe23cb27b3', sha256='f02b569f62a8d161a322625ba08b143bfb3ed4e19c8fead1ab6719747360c253', name='./rocksdb/000381.sst'), ManifestEntry(size=1089017411, md5='d428ab22dda7a2a40789ebc6c1da7610', sha256='4b14183a984a7d6d71fe353a47f1eb2c770baef152a5af55943d351d0a753a4a', name='./rocksdb/000382.sst'), ManifestEntry(size=1089188918, md5='105ba3e300a1d5b6c827ef6f55d3b7ed', sha256='30c2e153ba87da94897f01d0a21f1f5399700e07c63ead911cb1c5203a1ef276', name='./rocksdb/000383.sst'), ManifestEntry(size=1089056095, md5='94918cd025eb83cdc84bad6b507e0472', sha256='9a4213c8e3fee07c0573c17b5a863d777ad67f235d7a35444bd66379142cffff', name='./rocksdb/000384.sst'), ManifestEntry(size=1088813782, md5='41eac3bb8c3bc97902ce35503d257b9c', sha256='d73abdcc446ee53bd6d5d1f27b1bc9921d400698935f235e35bc2bf663c7684e', name='./rocksdb/000385.sst'), ManifestEntry(size=1088738712, md5='5d563b68c08ab3836e5ae3affd3d9f7e', sha256='eadd9cc501fcb805fb87be6757cae7e2f6765f06015b91b4529f34b44fe6376b', name='./rocksdb/000386.sst'), ManifestEntry(size=1089106381, md5='28ecc06fce9f7798a223e0dd2d57cb10', sha256='7cae5221dbae31dfc59046a64744210a1b696cb3128d7e7dadc4b65ec319a48f', name='./rocksdb/000387.sst'), ManifestEntry(size=1089262649, md5='e828f9d191d8941a3bd08ce88fc31f68', sha256='2da9074240ddca4786e913a8396963b1348f8714dff5d5fb05de26a5427a3a4c', name='./rocksdb/000388.sst'), ManifestEntry(size=1089302816, md5='9752708bbbc80821d189fe36aa2bac4c', sha256='f1d68d1877c1b0353e23849a20c66c6a093801c18f60a6a3aa383402c06daca1', name='./rocksdb/000389.sst'), ManifestEntry(size=1088110834, md5='44da39af9377477f119c4c7e722cf5c2', sha256='78228d63521281e575a92ad76116be2cf035e23dc96ab6b950fe81c43d5fda88', name='./rocksdb/000390.sst'), ManifestEntry(size=1089346071, md5='ffb30dd2f5b6a477a745272f940d4c05', sha256='7d65d5224f1c487977e7f5d6752b69c50cf0791eb066464cc1ab147c41bb317a', name='./rocksdb/000391.sst'), ManifestEntry(size=1089013161, md5='db805eda066735401cfa35c49892ffad', sha256='665bc665cd50ee5da39e39767ab835e59d9077f10e5e7957b7cdce60bad6635a', name='./rocksdb/000392.sst'), ManifestEntry(size=1088972734, md5='95b1147f1fd0fadec7185759c4f05883', sha256='0501794cbd24949f0fd404a463291540b4bae30a8fc511cce9cc9c735672da81', name='./rocksdb/000393.sst'), ManifestEntry(size=1089048629, md5='511c38c6497a2ee21f5dd6ab6c32fe3c', sha256='bc23c9dae5d7fab6188a812001923c67b6bc80c1b6c18227639c099f50549249', name='./rocksdb/000394.sst'), ManifestEntry(size=1089284704, md5='a70f9410b8d3e547174e747c1ed67866', sha256='b66f5a36654fdce09c538e3311e961e8c182be5dad3e876506f6048625fdc48c', name='./rocksdb/000395.sst'), ManifestEntry(size=1088982929, md5='a01527af1e94c731a7ec4eda10dca40f', sha256='cc46c8290d40899f08194a471b0acfaa452e55e2f3e47747fdc9e203847d3003', name='./rocksdb/000396.sst'), ManifestEntry(size=1089130285, md5='6b40b5a7af7e7345aab723d4e1cce064', sha256='82d2a3ed3ec01b2b18e418bcf73ec3c4498b06566a2bb08e5af1920bb3f0474e', name='./rocksdb/000397.sst'), ManifestEntry(size=1088976586, md5='0dee0be71fe191f4f412fafe49409c83', sha256='c81a00fe0a01e427f0bdd5dff58107cf595388c5c9d79b9e077e726dc911f675', name='./rocksdb/000398.sst'), ManifestEntry(size=1088855396, md5='278f0d888025a90d16672833abe81024', sha256='819ceaa0d29f5a0cd3084c1e1998247b2f8973c97fc64bc84d0903064df38853', name='./rocksdb/000399.sst'), ManifestEntry(size=1089260009, md5='e9832468d4353076c5c64195cfdab763', sha256='0185c3dab3c5c7b9c7e97321bc50b64872914f247f6e2d612d7bd0b2f6112699', name='./rocksdb/000400.sst'), ManifestEntry(size=1089257638, md5='c14092fa71dd9230acadf1834c729f2d', sha256='8619ad6b9ed4bac9f371d458a848e0584c5722248d68d8c9e1f532c43213697f', name='./rocksdb/000401.sst'), ManifestEntry(size=1089174502, md5='cc55451980cbc4e1a03e69df4addcdae', sha256='1ac43b27b2052c3c86466178628cb69c0d545b8e0f6de7209383274abe8d8a01', name='./rocksdb/000402.sst'), ManifestEntry(size=1089236809, md5='2ba7a96e58c0c4e3ec2fd1cb1a1fce44', sha256='2109784ca6bd3ef8d8b767d304df2dc046029e2f672a549553ade2a0216c3315', name='./rocksdb/000403.sst'), ManifestEntry(size=1089136948, md5='25f96529f975fcc5faae52928bb46d6e', sha256='854038cd91778e9a0c237938379c1a832d6a799030ff9bd7503fe8ff9ceaa14f', name='./rocksdb/000404.sst'), ManifestEntry(size=1089294821, md5='a48930cce95be74fc9e719de6506fa5c', sha256='5b815c2de5edac74d121be71f3e0df90083c78290413deec47504a39dd3fbb87', name='./rocksdb/000405.sst'), ManifestEntry(size=1089247949, md5='6291b6a5cf3f9c216e35b582ae82d389', sha256='572ce0ab8fae1b1a3fb95d37b2fe1395aff68837fb7d7b008fb4d2d2fdda6fdf', name='./rocksdb/000406.sst'), ManifestEntry(size=1089046034, md5='38dda484748b378eae5d48a44ffc8936', sha256='45e0dc396b19b75c7f64985373e920e22506205c8835d4a352465b61ba3d539c', name='./rocksdb/000407.sst'), ManifestEntry(size=1089138528, md5='61715bedec63061261485b583bf54d55', sha256='a3425e6dbabec000f790ec738e446046be6e868044d2ca92104a48bbe4ca8b7e', name='./rocksdb/000408.sst'), ManifestEntry(size=1088961607, md5='59ac19d1f3bf2f84f208454fcd5ec60f', sha256='693617869c19fbedbffa6832138b6dd661591e54e3fabea3a5363ce43e4d1017', name='./rocksdb/000409.sst'), ManifestEntry(size=1089276807, md5='62319ef6607ddc45b9d668140c8c7d36', sha256='6ffddafdb31ca481c6eace1c744f03af143d0c474c50b01739f993efaab4530e', name='./rocksdb/000410.sst'), ManifestEntry(size=1088962325, md5='7fb206d657d7b2382755fc1b8538ef5a', sha256='f4a3584ff1e9487ebc0fe9945014f307b2c4d0550da8acbeffa5ab5a4b1da1bc', name='./rocksdb/000411.sst'), ManifestEntry(size=1089115549, md5='c2a14a1a51e8a5540647dcec3947befe', sha256='22cf6bb0042d4b5e0833715d53c8d82acc9ad1b6c917f24e0961bbb02037358f', name='./rocksdb/000412.sst'), ManifestEntry(size=1089093235, md5='1e56706a6a72dc049a7df6ee0b85129a', sha256='0e2cb93f1df7d2db461c9789fd103234cab950aa4705f75a80be7dd4fbea9df6', name='./rocksdb/000413.sst'), ManifestEntry(size=1088971425, md5='d2539a82d2864bff326b6aadbff97d34', sha256='402519d9a92bba62a3b360ee2a10724a9d52763ce57f77a3f4c0016fd2e560ad', name='./rocksdb/000414.sst'), ManifestEntry(size=1089171186, md5='52cdb06cb054ae9140f4d49488113e48', sha256='74dfd3503a2ccf71b4b83077437102be50243bfadc01baf54a2ade49a70fb4d3', name='./rocksdb/000415.sst'), ManifestEntry(size=859126362, md5='a37af6fff97efd0d090a88b5964ddc79', sha256='dc5f82e43a20f0a71f46c0718393244dce1ea02d96dc2e5f06a33afba1dd7ce4', name='./rocksdb/000416.sst'), ManifestEntry(size=694644230, md5='04373b1020e2ed5c4fe3ef844f5a743b', sha256='8b8ab7838627a90e8cc9e52de5eb5b7668c7686c1741240c3e509e5805aeee52', name='./rocksdb/000417.sst'), ManifestEntry(size=879071264, md5='c82041866827852b3765833fcdb05792', sha256='353d33acf7e6fd8f6d6369d72c6abe474b233e11596db30d0e98065f6dacc4f7', name='./rocksdb/000418.sst'), ManifestEntry(size=895176213, md5='4648b340f43c32b8eecd5679ff43ab23', sha256='ed0c5056f190b98083ed7844c6b9c894be64e5e8e1d69f4f5a1d2a6f420fea36', name='./rocksdb/000419.sst'), ManifestEntry(size=862698256, md5='150c80987ef8729a00b55ecfa86a4f2a', sha256='430544b47b033a75bff5da664bd90ef7eeaf54e09d930cabddf96e168f785cd5', name='./rocksdb/000420.sst'), ManifestEntry(size=884488381, md5='330809257229ed35ada44952c52f4828', sha256='bcf01a45623a9e2f6f53410a69681f9b31cbd1f0d0ec956520b27095734cde1a', name='./rocksdb/000421.sst'), ManifestEntry(size=910629924, md5='d7e016dfed4f2726b01b152eee37b41d', sha256='590b5090a0aea06ad78a83049fd299d00b245ad7cb30f8b677fcc0eec67d41c9', name='./rocksdb/000422.sst'), ManifestEntry(size=37395883, md5='8d950020bb50e1fc1b4e619f23051208', sha256='aab8b81c56b26dfb3641bb689a3d07b5e76b942e23939ed1b55ba29d39a86eea', name='./rocksdb/000423.sst'), ManifestEntry(size=16, md5='9f36605efba98dab15728fe8b5538aa0', sha256='9c283f6e81028b9eb0760d918ee4bc0aa256ed3b926393c1734c760c4bd724fd', name='./rocksdb/CURRENT'), ManifestEntry(size=36, md5='053182ada9c50b2d8010935c6da8b1b4', sha256='1dc7b40d8645ad5084cf8ff0c483048e78be7739f3861456ca31e42e833b539d', name='./rocksdb/IDENTITY'), ManifestEntry(size=0, md5='d41d8cd98f00b204e9800998ecf8427e', sha256='e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855', name='./rocksdb/LOCK'), ManifestEntry(size=1060894, md5='091faf0b2b35346ac07368aaf3b1fae6', sha256='dc917ab03b15659b4eec772320478d038e2a68d27dfc77eeadcbc942524a100d', name='./rocksdb/LOG'), ManifestEntry(size=31518, md5='952ef0c700c1f640f95db43d42c2ff61', sha256='350e2556fa2bf105c5e8abf3cf7343418d04f90ae11ec4a7a3f92b312d0239eb', name='./rocksdb/MANIFEST-000005'), ManifestEntry(size=15392, md5='25c6dbe82754022c1d0985514cc2c4dd', sha256='e1478633f327f9ddfaa608bd11e263bfc45d9fde5b613cdd2a7eefe0251cf486', name='./rocksdb/OPTIONS-000009'), ManifestEntry(size=15392, md5='25c6dbe82754022c1d0985514cc2c4dd', sha256='e1478633f327f9ddfaa608bd11e263bfc45d9fde5b613cdd2a7eefe0251cf486', name='./rocksdb/OPTIONS-000011')])
# ---
